Upgrade EUI to v39.1.1

cee-chen opened the pull request:
## Summary

`eui@39.0.0` ⏩ `eui@39.1.1`
___

## [`39.1.1`](https://github.com/elastic/eui/tree/v39.1.1)

**Bug fixes**

- Fixed `SuperDatePicker` from crashing due to invalid time input ([#5263](https://github.com/elastic/eui/pull/5263))
- Fixed content in `EuiFilterButton` again when `numFilters` is undefined ([#5268](https://github.com/elastic/eui/pull/5268))

## [`39.1.0`](https://github.com/elastic/eui/tree/v39.1.0)

- Added support for `ghost` and `text` `EuiIcon` colors on Elastic logos ([#5245](https://github.com/elastic/eui/pull/5245))
- Added a default `data-test-subj` to `EuiErrorBoundary` ([#5232](https://github.com/elastic/eui/pull/5232))

**Bug fixes**

- Fixed content in `EuiPopover` from being inaccessible during the opening animation ([#5249](https://github.com/elastic/eui/pull/5249))
- Fixed multiple accessibility issues in `EuiBasicTable` and `EuiInMemoryTable` ([#5241](https://github.com/elastic/eui/pull/5241))

cee-chen committed:
Upversion to EUI 39.1.0

cee-chen committed:
Update i18n_eui_mapping tokens

@see https://github.com/elastic/eui/blob/master/i18ntokens_changelog.json

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160245/)
* Commit: 49426450d4de9c375104ae7887e9c5e1946a89bc
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-114732/49426450d4de9c375104ae7887e9c5e1946a89bc)
* This comment will update when the build is complete

### Failed CI Steps
* [Jest Unit Tests](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160245/execution/node/190/log)

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160245/","number":"160245","commit":"49426450d4de9c375104ae7887e9c5e1946a89bc"}]}
PIPELINE-->

cee-chen committed:
Merge refractor in yarn.lock

cee-chen committed:
Fix functional table filter selector

- Popover ID was removed in recent EUI a11y fix, so we're using child-position selection to target the Tags filter now

cee-chen committed:
Update snaphots

cee-chen committed:
Merge branch 'master' into eui-39.1.0

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160683/)
* Commit: 0c60bd41383b701c990cc0cc7202fffe5969816f
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-114732/0c60bd41383b701c990cc0cc7202fffe5969816f)
* This comment will update when the build is complete

### Failed CI Steps
* [Execute xpack-securitySolutionCypressChrome](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160683/execution/node/548/log)
* [Execute xpack-securitySolutionCypressChrome](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160683/execution/node/1094/log)

### History
* :broken_heart: [Build #160582](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160582/) failed 9fa84bd1df18ac925c9b1b6e2c7b32d0feebde57
* :broken_heart: [Build #160245](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160245/) failed 49426450d4de9c375104ae7887e9c5e1946a89bc

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160245/","number":"160245","commit":"49426450d4de9c375104ae7887e9c5e1946a89bc"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160582/","number":"160582","commit":"9fa84bd1df18ac925c9b1b6e2c7b32d0feebde57"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160683/","number":"160683","commit":"0c60bd41383b701c990cc0cc7202fffe5969816f"}]}
PIPELINE-->

cee-chen committed:
Upgrade to 39.1.1 for extra bugfixes

cee-chen committed:
Update i18n mappings

cee-chen committed:
Merge branch 'master' into eui-39.1.0

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160745/)
* Commit: cc5a35be3192a2ad3c71c305c1bd6c60a3422031
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-114732/cc5a35be3192a2ad3c71c305c1bd6c60a3422031)
* This comment will update when the build is complete

### Failed CI Steps
* [Jest Unit Tests](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160745/execution/node/190/log)

### History
* :broken_heart: [Build #160683](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160683/) failed 0c60bd41383b701c990cc0cc7202fffe5969816f
* :broken_heart: [Build #160582](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160582/) failed 9fa84bd1df18ac925c9b1b6e2c7b32d0feebde57
* :broken_heart: [Build #160245](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160245/) failed 49426450d4de9c375104ae7887e9c5e1946a89bc

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160245/","number":"160245","commit":"49426450d4de9c375104ae7887e9c5e1946a89bc"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160582/","number":"160582","commit":"9fa84bd1df18ac925c9b1b6e2c7b32d0feebde57"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160683/","number":"160683","commit":"0c60bd41383b701c990cc0cc7202fffe5969816f"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160745/","number":"160745","commit":"cc5a35be3192a2ad3c71c305c1bd6c60a3422031"}]}
PIPELINE-->

cee-chen committed:
Merge branch 'master' into eui-39.1.0

cee-chen committed:
Fix i18n snapshot

kibanamachine committed:
Merge branch 'master' into eui-39.1.0

kibanamachine committed:
Merge branch 'master' into eui-39.1.0

cee-chen committed:
Attempt to harden flaky Security Cypress test

cee-chen committed:
More combobox entry hardening

- Got a flake on clicking the combobox dropdown on run 17/20 locally

kibanamachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/161143/)
* Commit: c8b30c1f4796d87f2a40225785c520cc5a8c3961
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-114732/c8b30c1f4796d87f2a40225785c520cc5a8c3961)
* [Documentation Changes](https://kibana_114732.docs-preview.app.elstc.co/diff)
* Flaky suites:
  * xpack-kibana-ciGroup13

---

### [Test Failures](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/161143/testReport)

<details><summary>Kibana Pipeline / general / License plugin API Integration Tests.x-pack/test/licensing_plugin/server/updates·ts.Licensing plugin server client changes in license types provides changes in license types</summary>

[Link to Jenkins](https://kibana-ci.elastic.co//job/elastic+kibana+pipeline-pull-request/161143/testReport/junit/License%20plugin%20API%20Integration%20Tests/x-pack_test_licensing_plugin_server_updates·ts/Kibana_Pipeline___general___Licensing_plugin_server_client_changes_in_license_types_provides_changes_in_license_types)


#### Standard Out
```
Failed Tests Reporter:
  - Test has failed 1 times on tracked branches: https://github.com/elastic/kibana/issues/110938

[00:00:00]     │
[00:00:00]       └-: Licensing plugin server client
[00:00:00]         └-> "before all" hook in "Licensing plugin server client"
[00:00:00]         └-: changes in license types
[00:00:00]           └-> "before all" hook for "provides changes in license types"
[00:00:00]           └-> provides changes in license types
[00:00:00]             └-> "before each" hook: global before each for "provides changes in license types"
[00:00:00]             │ debg creating role license_manager-role
[00:00:00]             │ info [o.e.x.s.a.r.TransportPutRoleAction] [node-01] added role [license_manager-role]
[00:00:00]             │ debg creating user license_manager_user
[00:00:00]             │ info [o.e.x.s.a.u.TransportPutUserAction] [node-01] added user [license_manager_user]
[00:00:00]             │ debg created user license_manager_user
[00:00:00]             │ debg SecurityPage.logout
[00:00:00]             │ debg TestSubjects.exists(userMenuButton)
[00:00:00]             │ debg Find.existsByDisplayedByCssSelector('[data-test-subj="userMenuButton"]') with timeout=2500
[00:00:03]             │ debg --- retry.tryForTime error: [data-test-subj="userMenuButton"] is not displayed
[00:00:03]             │ debg Logout not found
[00:00:03]             │ debg TestSubjects.exists(loginForm)
[00:00:03]             │ debg Find.existsByDisplayedByCssSelector('[data-test-subj="loginForm"]') with timeout=2500
[00:00:06]             │ debg --- retry.tryForTime error: [data-test-subj="loginForm"] is not displayed
[00:00:06]             │ debg navigating to login url: http://localhost:6141/login
[00:00:06]             │ debg navigate to: http://localhost:6141/login
[00:00:07]             │ debg browser[INFO] http://localhost:6141/login?_t=1634245740033 281 Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'unsafe-eval' 'self'". Either the 'unsafe-inline' keyword, a hash ('sha256-P5polb1UreUSOe5V/Pv7tc+yeZuJXiOi/3fqhGsU7BE='), or a nonce ('nonce-...') is required to enable inline execution.
[00:00:07]             │
[00:00:07]             │ debg browser[INFO] http://localhost:6141/bootstrap.js 41:19 "^ A single error about an inline script not firing due to content security policy is expected!"
[00:00:07]             │ debg ... sleep(700) start
[00:00:07]             │ debg ... sleep(700) end
[00:00:07]             │ debg returned from get, calling refresh
[00:00:09]             │ debg browser[INFO] http://localhost:6141/login?_t=1634245740033 281 Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'unsafe-eval' 'self'". Either the 'unsafe-inline' keyword, a hash ('sha256-P5polb1UreUSOe5V/Pv7tc+yeZuJXiOi/3fqhGsU7BE='), or a nonce ('nonce-...') is required to enable inline execution.
[00:00:09]             │
[00:00:09]             │ debg browser[INFO] http://localhost:6141/bootstrap.js 41:19 "^ A single error about an inline script not firing due to content security policy is expected!"
[00:00:09]             │ debg currentUrl = http://localhost:6141/login
[00:00:09]             │          appUrl = http://localhost:6141/login
[00:00:09]             │ debg TestSubjects.find(kibanaChrome)
[00:00:09]             │ debg Find.findByCssSelector('[data-test-subj="kibanaChrome"]') with timeout=60000
[00:00:10]             │ debg ... sleep(501) start
[00:00:11]             │ warn browser[SEVERE] http://localhost:6141/api/licensing/info - Failed to load resource: the server responded with a status of 401 (Unauthorized)
[00:00:11]             │ debg ... sleep(501) end
[00:00:11]             │ debg in navigateTo url = http://localhost:6141/login
[00:00:11]             │ debg Waiting for Login Form to appear.
[00:00:11]             │ debg Waiting up to 100000ms for login form...
[00:00:11]             │ debg TestSubjects.exists(loginForm)
[00:00:11]             │ debg Find.existsByDisplayedByCssSelector('[data-test-subj="loginForm"]') with timeout=2500
[00:00:11]             │ debg TestSubjects.setValue(loginUsername, license_manager_user)
[00:00:11]             │ debg TestSubjects.click(loginUsername)
[00:00:11]             │ debg Find.clickByCssSelector('[data-test-subj="loginUsername"]') with timeout=10000
[00:00:11]             │ debg Find.findByCssSelector('[data-test-subj="loginUsername"]') with timeout=10000
[00:00:11]             │ debg TestSubjects.setValue(loginPassword, license_manager_user-password)
[00:00:11]             │ debg TestSubjects.click(loginPassword)
[00:00:11]             │ debg Find.clickByCssSelector('[data-test-subj="loginPassword"]') with timeout=10000
[00:00:11]             │ debg Find.findByCssSelector('[data-test-subj="loginPassword"]') with timeout=10000
[00:00:12]             │ debg TestSubjects.click(loginSubmit)
[00:00:12]             │ debg Find.clickByCssSelector('[data-test-subj="loginSubmit"]') with timeout=10000
[00:00:12]             │ debg Find.findByCssSelector('[data-test-subj="loginSubmit"]') with timeout=10000
[00:00:12]             │ debg Waiting for login result, expected: undefined.
[00:00:12]             │ debg Waiting up to 20000ms for logout button visible...
[00:00:12]             │ debg TestSubjects.exists(userMenuButton)
[00:00:12]             │ debg Find.existsByDisplayedByCssSelector('[data-test-subj="userMenuButton"]') with timeout=2500
[00:00:12]             │ proc [kibana] [2021-10-14T21:09:05.315+00:00][INFO ][plugins.security.routes] Logging in with provider "basic" (basic)
[00:00:14]             │ debg --- retry.tryForTime error: [data-test-subj="userMenuButton"] is not displayed
[00:00:15]             │ debg TestSubjects.exists(userMenuButton)
[00:00:15]             │ debg Find.existsByDisplayedByCssSelector('[data-test-subj="userMenuButton"]') with timeout=2500
[00:00:17]             │ debg browser[INFO] http://localhost:6141/app/home 281 Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'unsafe-eval' 'self'". Either the 'unsafe-inline' keyword, a hash ('sha256-P5polb1UreUSOe5V/Pv7tc+yeZuJXiOi/3fqhGsU7BE='), or a nonce ('nonce-...') is required to enable inline execution.
[00:00:17]             │
[00:00:17]             │ debg browser[INFO] http://localhost:6141/bootstrap.js 41:19 "^ A single error about an inline script not firing due to content security policy is expected!"
[00:00:18]             │ debg TestSubjects.exists(userMenu)
[00:00:18]             │ debg Find.existsByDisplayedByCssSelector('[data-test-subj="userMenu"]') with timeout=2500
[00:00:18]             │ info [o.e.c.m.MetadataMappingService] [node-01] [.kibana_8.0.0_001/jjtduFBFT7yFkAk0VRN5ag] update_mapping [_doc]
[00:00:18]             │ info [o.e.x.i.IndexLifecycleTransition] [node-01] moving index [.ds-ilm-history-5-2021.10.14-000001] from [{"phase":"new","action":"complete","name":"complete"}] to [{"phase":"hot","action":"unfollow","name":"branch-check-unfollow-prerequisites"}] in policy [ilm-history-ilm-policy]
[00:00:18]             │ info [o.e.x.i.IndexLifecycleTransition] [node-01] moving index [.ds-ilm-history-5-2021.10.14-000001] from [{"phase":"hot","action":"unfollow","name":"branch-check-unfollow-prerequisites"}] to [{"phase":"hot","action":"rollover","name":"check-rollover-ready"}] in policy [ilm-history-ilm-policy]
[00:00:20]             │ debg --- retry.tryForTime error: [data-test-subj="userMenu"] is not displayed
[00:00:21]             │ debg TestSubjects.click(userMenuButton)
[00:00:21]             │ debg Find.clickByCssSelector('[data-test-subj="userMenuButton"]') with timeout=10000
[00:00:21]             │ debg Find.findByCssSelector('[data-test-subj="userMenuButton"]') with timeout=10000
[00:00:21]             │ debg TestSubjects.exists(userMenu)
[00:00:21]             │ debg Find.existsByDisplayedByCssSelector('[data-test-subj="userMenu"]') with timeout=120000
[00:00:21]             │ debg TestSubjects.exists(userMenu > logoutLink)
[00:00:21]             │ debg Find.existsByDisplayedByCssSelector('[data-test-subj="userMenu"] [data-test-subj="logoutLink"]') with timeout=2500
[00:00:22]             │ info [o.e.l.LicenseService] [node-01] license [e3d7573c-41f1-4e05-b1ef-d19ed9e23b86] mode [trial] - valid
[00:00:22]             │ info [o.e.x.s.a.Realms] [node-01] license mode is [trial], currently licensed security realms are [reserved/reserved,file/default_file,native/default_native]
[00:00:24]             │ proc [kibana] [2021-10-14T21:09:17.538+00:00][INFO ][plugins.monitoring.monitoring.kibana-monitoring] Starting monitoring stats collection
[00:00:24]             │ info Taking screenshot "/dev/shm/workspace/parallel/4/kibana/x-pack/test/licensing_plugin/screenshots/failure/Licensing plugin server client changes in license types provides changes in license types.png"
[00:00:24]             │ info Current URL is: http://localhost:6141/app/home#/
[00:00:24]             │ info Saving page source to: /dev/shm/workspace/parallel/4/kibana/x-pack/test/licensing_plugin/failure_debug/html/Licensing plugin server client changes in license types provides changes in license types.html
[00:00:24]             └- ✖ fail: Licensing plugin server client changes in license types provides changes in license types
[00:00:24]             │      Error: expected 'basic' to equal 'trial'
[00:00:24]             │       at Assertion.assert (/dev/shm/workspace/parallel/4/kibana/node_modules/@kbn/expect/expect.js:100:11)
[00:00:24]             │       at Assertion.be.Assertion.equal (/dev/shm/workspace/parallel/4/kibana/node_modules/@kbn/expect/expect.js:227:8)
[00:00:24]             │       at Assertion.be (/dev/shm/workspace/parallel/4/kibana/node_modules/@kbn/expect/expect.js:69:22)
[00:00:24]             │       at Context.<anonymous> (test/licensing_plugin/server/updates.ts:45:45)
[00:00:24]             │       at runMicrotasks (<anonymous>)
[00:00:24]             │       at processTicksAndRejections (internal/process/task_queues.js:95:5)
[00:00:24]             │       at Object.apply (/dev/shm/workspace/parallel/4/kibana/node_modules/@kbn/test/target_node/functional_test_runner/lib/mocha/wrap_function.js:87:16)
[00:00:24]             │ 
[00:00:24]             │ 

```

#### Stack Trace
```
Error: expected 'basic' to equal 'trial'
    at Assertion.assert (/dev/shm/workspace/parallel/4/kibana/node_modules/@kbn/expect/expect.js:100:11)
    at Assertion.be.Assertion.equal (/dev/shm/workspace/parallel/4/kibana/node_modules/@kbn/expect/expect.js:227:8)
    at Assertion.be (/dev/shm/workspace/parallel/4/kibana/node_modules/@kbn/expect/expect.js:69:22)
    at Context.<anonymous> (test/licensing_plugin/server/updates.ts:45:45)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at Object.apply (/dev/shm/workspace/parallel/4/kibana/node_modules/@kbn/test/target_node/functional_test_runner/lib/mocha/wrap_function.js:87:16)
```
</details>

---

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/35da3233130eccb938568f298c602f5829742d6c) | [after](https://github.com/elastic/kibana/commit/c8b30c1f4796d87f2a40225785c520cc5a8c3961) | diff |
| --- | --- | --- | --- |
| `core` | 132.4KB | 132.4KB | -1.0B |
| `fleet` | 635.1KB | 635.1KB | -1.0B |
| `kibanaReact` | 203.5KB | 203.5KB | -1.0B |
| `kibanaUtils` | 52.2KB | 52.2KB | -1.0B |
| `security` | 514.6KB | 514.6KB | -1.0B |
| `securitySolution` | 4.6MB | 4.6MB | -1.0B |
| total |  |  | -6.0B |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/35da3233130eccb938568f298c602f5829742d6c) | [after](https://github.com/elastic/kibana/commit/c8b30c1f4796d87f2a40225785c520cc5a8c3961) | diff |
| --- | --- | --- | --- |
| `core` | 302.8KB | 302.9KB | +118.0B |
| `kbnUiSharedDeps-css` | 597.0KB | 597.0KB | +27.0B |
| `kbnUiSharedDeps-npmDll` | 5.1MB | 5.1MB | +819.0B |
| total |  |  | +964.0B |

### History
* :green_heart: [Build #161098](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/161098/) succeeded 0b95ef7367be41a38afbe9cee13fe2b634b9ef97
* :broken_heart: [Build #161013](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/161013/) failed 7f2388cf8da083c1393766f296342d1dae0333bd
* :broken_heart: [Build #160764](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160764/) failed bcad676dbba2825b2556472b177681a783c0a79b
* :broken_heart: [Build #160745](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160745/) failed cc5a35be3192a2ad3c71c305c1bd6c60a3422031
* :broken_heart: [Build #160683](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160683/) failed 0c60bd41383b701c990cc0cc7202fffe5969816f

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160683/","number":"160683","commit":"0c60bd41383b701c990cc0cc7202fffe5969816f"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160745/","number":"160745","commit":"cc5a35be3192a2ad3c71c305c1bd6c60a3422031"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/160764/","number":"160764","commit":"bcad676dbba2825b2556472b177681a783c0a79b"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/161013/","number":"161013","commit":"7f2388cf8da083c1393766f296342d1dae0333bd"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/161098/","number":"161098","commit":"0b95ef7367be41a38afbe9cee13fe2b634b9ef97"},{"status":"UNSTABLE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/161143/","number":"161143","commit":"c8b30c1f4796d87f2a40225785c520cc5a8c3961"}]}
PIPELINE-->

justinkambic reviewed (approved):
Uptime changes LGTM

smith reviewed (approved):
APM looks good

kibanamachine committed:
Merge branch 'master' into eui-39.1.0

constancecchen commented:
@elastic/kibana-app-services @elastic/kibana-presentation @elastic/kibana-reporting-services @elastic/security-solution I'm hoping to merge this by EOD in time for 7.16 FF - would appreciate your review before then.

ThomThomson reviewed (approved):
Great to see https://github.com/elastic/eui/pull/5268 included, as that will be needed for the Controls project. Presentation team changes LGTM!

kibanamachine committed:
Merge branch 'master' into eui-39.1.0

constancecchen commented:
@elasticmachine merge upstream

kibanamachine committed:
Merge branch 'master' into eui-39.1.0

constancecchen commented:
@elasticmachine merge upstream

kibanamachine committed:
Merge branch 'master' into eui-39.1.0

kibanamachine commented:
## 💚 Backport successful

| Status | Branch | Result |
|:------:|:------:|:------:|
| ✅ |  [7.x](https://github.com/elastic/kibana/pull/115598)  | [<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/115598">](https://github.com/elastic/kibana/pull/115598) |

This backport PR will be merged automatically after passing CI.
